crates/wit-parser/src/ast/resolve.rs

```diff
@@ -141,14 +184,78 @@ impl Resolver {
         }
     }
 
+    fn process_use(&mut self, fields: &[InterfaceItem<'_>]) -> Result<()> {
+        for item in fields.iter() {
+            let u = match item {
+                InterfaceItem::Use(u) => u,
+                _ => continue,
+            };
+            let _dep_id = &self.interface_lookup[&*u.from.name];
+            let interface_name = &u.from.name;
+            let from_span = u.from.span;
+            match &u.names {
+                Some(names) => {
+                    for name in names {
+                        let (my_name, span) = match &name.as_ {
+                            Some(id) => (&id.name, id.span),
+                            None => (&name.name.name, name.name.span),
+                        };
+                        let mut found = false;
```

Comment:
Instead of a `found` bool here I think the `None` case can be matched on the `qualified_type_names.get` below I think?